src/devices/efekta.ts

```diff
@@ -762,6 +769,372 @@ const definitions: DefinitionWithExtend[] = [
+            }),
+            numeric({
+                endpointNames: ['2'],
+                name: 'raw_temperature_cal',
+                unit: 'raw unit',
+                valueMin: -8192,
+                valueMax: 8192,
+                cluster: 'msTemperatureMeasurement',
+                attribute: {ID: 0x0210, type: 0x29},
+                description: 'Adjust second temperature sensor',
+            }),
+            battery({
+                voltage: true,
+                voltageReporting: true,
+                lowStatus: true,
+                percentageReportingConfig: fiveReporting,
+                voltageReportingConfig: sixReporting,
+            }),
+            numeric({
+                name: 'uptime',
+                unit: 'Hours',
+                cluster: 'genTime',
+                attribute: 'localTime',
+                description: 'Uptime',
+                scale: 3600,
+                precision: 1,
+                access: 'STATE',
+            }),
+            numeric({
+                name: 'reading_interval',
+                unit: 'sec',
+                valueMin: 1,
+                valueMax: 360,
+                cluster: 'genPowerCfg',
+                attribute: {ID: 0x0201, type: 0x21},
+                description: 'Setting the sensor reading interval in seconds, by default 15 seconds',
+            }),
+            enumLookup({
+                name: 'tx_radio_power',
+                lookup: {'0': 0, '4': 4},
+                cluster: 'genPowerCfg',
+                attribute: {ID: 0x0236, type: 0x28},
+                description: 'Set TX Radio Power, dbm',
+            }),
+            binary({
+                name: 'pool_rate_on',
+                valueOn: ['ON', 1],
+                valueOff: ['OFF', 0],
+                cluster: 'genPowerCfg',
+                attribute: {ID: 0x0216, type: 0x10},
+                description: 'Enable Smart Sleep, short wakeup every 7 seconds',
+            }),
+            binary({
+                name: 'config_report_enable',
+                valueOn: ['ON', 1],
+                valueOff: ['OFF', 0],
+                cluster: 'genPowerCfg',
+                attribute: {ID: 0x0275, type: 0x10},
+                description: 'Enable reporting based on reporting configuration',
+            }),
+            binary({
+                name: 'comparison_previous_data',
+                valueOn: ['ON', 1],
+                valueOff: ['OFF', 0],
+                cluster: 'genPowerCfg',
+                attribute: {ID: 0x0205, type: 0x10},
+                description: 'Enable сontrol of comparison with previous data',
+            }),
+        ],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpointOne = device.getEndpoint(1);
+            const overrides0 = {min: 7200, max: 43200, change: 1};
+            await reporting.batteryAlarmState(endpointOne, overrides0);
```

Comment:
I think this can also be added as a modernExtend?

Reply:
It would be great, as long as it doesn't work through modernity. As soon as you implement it, I will add efekta.ts

Reply:
Could you check if you can add this? Add `batteryAlarmStateReporting` to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/lib/modernExtend.ts#L285